### PR TITLE
Branch student enroll command

### PR DESCRIPTION
**Pull request by LuYiting0913**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/student/StudentEditCommand.java [hunk 1]
```diff
@@ -88,11 +88,6 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(MESSAGE_DUPLICATE_PERSON);
         }
 
-        if (studentToEdit.belongsTo(editedStudent.getTutorialGroup())
-                && !model.hasTutorialGroup(editedStudent.getTutorialGroup())) {
-            throw new CommandException(MESSAGE_TUTORIAL_GROUP_NOT_FOUND);
-        }
-
```
Comment (gunbux) [inline src/main/java/seedu/address/logic/commands/student/StudentEditCommand.java:95]: We should probably change the help blurb for edit command also if we're removing this

File: src/main/java/seedu/address/logic/commands/task/TaskEditCommand.java [hunk 2]
```diff
@@ -77,6 +77,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Task taskToEdit = lastShownList.get(index.getZeroBased());
+        System.out.println("---");
```
Comment (gunbux) [inline src/main/java/seedu/address/logic/commands/task/TaskEditCommand.java]: wemove pweese OWO

**Review by gunbux — COMMENTED** [review]
smol changes uwu otherwise looks good owo

File: src/main/java/seedu/address/logic/commands/task/TaskEditCommand.java (hunk 2 above)
Comment (yeojunjie) [reply src/main/java/seedu/address/logic/commands/task/TaskEditCommand.java]:
```suggestion
```

File: src/main/java/seedu/address/logic/commands/student/StudentEnrollCommand.java [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+    public static final String COMMAND_WORD = "studentEnroll";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Enroll the student identified to the given tutorial "
+            + "by the index number used in the displayed student list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_TUTORIAL_GROUP + "TUTORIAL GROUP ";
+
+    public static final String MESSAGE_ENROLL_PERSON_SUCCESS = "Enrolled Student to: %1$s ";
```
Comment (yeojunjie) [inline src/main/java/seedu/address/logic/commands/student/StudentEnrollCommand.java:38]: Would it be better to name this constant `MESSAGE_ENROLL_STUDENT_SUCCESS`?

File: src/main/java/seedu/address/logic/commands/student/StudentEnrollCommand.java [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    public static final String MESSAGE_DUPLICATE_PERSON = "This student already exists in the address book.";
```
Comment (yeojunjie) [inline src/main/java/seedu/address/logic/commands/student/StudentEnrollCommand.java:39]: Would it be better to name this constant `MESSAGE_DUPLICATE_STUDENT`?

**Review by yeojunjie — APPROVED** [review]

**Pull request merged**